lib/cluster/workers/IDCardRenewer.js

```diff
@@ -29,6 +29,10 @@ class IDCardRenewer {
       const redisConf = config.redis || {};
       await this.initRedis(redisConf.config, redisConf.name);
     } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(
+        `Failed to connect to redis, could not refresh ID card: ${error.message}`
+      );
```

Comment:
Needed because it in a separeted worker_thread as @Shiranuit explained to me

Reply:
It's intended since there is no logger in the fork process this log is here to let us know of an issue happening inside the fork, even though it should be able to send the information back to the main process, in case something prevents it from notifying the main process there will also be a log from the fork.